rhel7/kickstart/ssg-rhel7-anssi_nt28_enhanced-ks.cfg

```diff
@@ -99,7 +99,7 @@ zerombr
 clearpart --linux --initlabel
 
 # Create primary system partitions (required for installs)
-part /boot --fstype=xfs --size=512
+part /boot --fstype=xfs --size=512 --fsoptions="nodev"
```

Comment:
What do you think of setting all recommended options for `/boot`? Which are `nosuid,nodev,noexec`.
All other configured partitions have them all.

Reply:
Sure, I will add them.